styles/aboutUs.scss

```diff
@@ -3,81 +3,123 @@
   width: 100%;
   width: 100vw;
 
-  padding-top: 100px;
-  margin-bottom: 100px;
+  padding-top: $spc-l;
+  padding-bottom: $spc-xl;
 
   flex-flow: column nowrap;
   align-items: center;
   text-align: center;
-  color: $font-color-dark;
-
-  background-image: url(assets/images/astronaut.svg);
-  background-size: 300px 300px;
-  background-position: 100% 20%;
-  background-repeat: no-repeat;
+  color: $font-color;
 
   .about-us-header {
     display: flex;
-    width: 100%;
-    width: 100vw;
-    margin-bottom: 60px;
-    flex-flow: column nowrap;
+    width: 75px;
+    width: 75vw;
+
+    margin-bottom: 100px;
```

Comment:
This as well

Reply:
Maybe better solution is to replace it with variable $spc-l, and then replace px with rem in variable value.

Reply:
Sure, that sounds good 👍